README.md

```diff
@@ -18,7 +18,8 @@ Build the back-end server using `maven install` with the supplied `pom.xml` file
 
 ### Deployable zip
 #####Unix systems:
-Run the `install.sh` script, this generates a zip file, extract this zip file to the desired location on your system. The servers can then be started by executing the `install.sh` script.
+Run the `install.sh` script, this generates a zip file, extract this zip file to the desired location on your system. The servers can then be started by executing the `startserver.sh` script. If maven is not installed on your system, run the `other systems` install instructions. 
+
```

Comment:
Oh I was mistaken when editing this description. 
If we distribute a zip in the release, the install instructions are not even needed. If it is zipped correctly it should run out of the box.